### ISSUE TITLE
Manage resolvconf service reloading

### ISSUE THREAD

**Issue by pataquets** [thias/puppet-resolvconf#4]
Under Ubuntu 12.04 Precise, resolv.conf file is managed by the resolvocnf service and package.
After resolv.conf, /etc/resolvconf/resolv.conf.d/{head,tail} modifications, the sevrice must be refreshed for changes to take effect.
I'm not sure if this is Ubuntu-specific or it might apply to other non-Debian distros.

**Comment by thias** [comment]
Could you look at my service branch, especially commit e3f2d57d3f28e01d92e28421c0960c7cc0be862a, and let me know if that's what you wanted? (note that I don't have any Ubuntu systems handy, so it's completely untested)

**Comment by thias** [comment]
Ping? Still interested in this being included?

**Comment by pataquets** [comment]
Yes, thanks. I'm still interested but could not get my hands on any puppet-managed desktop nor had time enough to test it. I'll try to find time to test it but can't tell when, sorry.
